se.cendio.tlclient.json

```diff
@@ -0,0 +1,58 @@
+{
+	"app-id": "se.cendio.tlclient",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "21.08",
+	"sdk": "org.freedesktop.Sdk",
+	"command": "tlclient-openconf",
+	"tags": [
+		"proprietary"
+	],
+	"finish-args": [
+		"--persist=.fltk",
+		"--persist=.thinlinc",
+		"--share=ipc",
+		"--share=network",
+		"--socket=x11",
+		"--socket=pulseaudio"
```

Comment:
How about Kerberos support?

Reply:
I assume krb5 and basic config needs bundling like Chromium does: https://github.com/flathub/org.chromium.Chromium/blob/master/org.chromium.Chromium.yaml#L112

Reply:
…and this needs adding too, probably? https://github.com/flathub/org.chromium.Chromium/blob/master/org.chromium.Chromium.yaml#L9

Reply:
I could add this, but I'm not sure how to test whether it is working. Once we have screenshots the bot will produce a test build you can try though, @CendioOssman.

Reply:
Sure. Kerberos is a bit of a hassle to set up so we can have a look once there is a test build.

Reply:
I'm afraid it does not work. So the app needs to include at least krb5-libs (specifically `libgssapi_krb5.so.2`, no idea about configuration).

Reply:
libgssapi_krb5.so.2 should now be present.